tests/ptable/plotly/test_ptable_heatmap_plotly.py

```
@@ -292,7 +292,7 @@ def test_ptable_heatmap_plotly_hover_props() -> None:
 
 def test_ptable_heatmap_plotly_custom_label_map() -> None:
     values = {"Fe": 2, "O": 3, "H": np.nan}
-    label_map = lambda label: {2: "High", 3: "Low"}.get(float(label), label)
+    label_map = lambda label: {2.0: "High", 3.0: "Low"}.get(float(label), label)
```

Ruff (0.15.12)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `label_map` as a `def`

(E731)